src/ergw_aaa_radius.erl

```diff
@@ -359,8 +363,12 @@ vendor_ituma('WLAN-Station-RSSI ', #{'WLAN-Station-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Sta_RSSI', Value}|Attrs];
 vendor_ituma('WLAN-AP-RSSI', #{'WLAN-AP-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Wwan_RSSI', Value}|Attrs];
+vendor_ituma('LI-Location', #{'IM_LI_Location' := LocationStr}, Attrs) ->
+    ?LOG(debug, "Adding LI-Location (decimal string): ~s", [LocationStr]),
+    [{?'IM_LI_Location', LocationStr}|Attrs];
```

Comment:
```suggestion
    [{?'IM_LI_Location', LocationStr} | Attrs];
```

Reply:
ok

Reply:
that is how it was done initially above, does it has sense to change it here (or on only single place)?

Reply:
Hm... it looks like it used mixed format in the module. Yes this can be skipped.

Reply:
ok